package.json

```diff
@@ -3,22 +3,17 @@
   "version": "1.0.0",
   "description": "Monorepo related to the GemWallet extension",
   "private": true,
+  "repository": "https://github.com/GemWallet/gemwallet-extension.git",
```

Comment:
Why isn't it an object anymore?

Reply:
It was generated like this when I set up the repo